src/main/java/com/fundation/logic/common/MetadataImageExtractor.java

```diff
@@ -60,39 +60,41 @@ public void readAll() {
 
     /**
      * This method read mime type.
-     *
-     * @return
+     * @return void
      */
     public static void mimeType(String mimeType) {
-        String imageMimeType = mimeType.substring(0,9);
-        if ((imageMimeType.contains("MIME Type"))) {
-            if (mimeType.contains("image")) {
-                searchMimeType = "image";
+        String mimetype2 = mimeType;
```

Comment:
Define more significant name for mimetype2 variable, for example, auxiliarMimetype or another.